src/main/resources/application.yml

```
@@ -23,7 +23,7 @@ spring:
 # Fake User #
 #############
 fake-user:
-  enabled: true
+  enabled: false
```

```suggestion
  enabled: true
```

@MiniDigger Do you want this to still default to `true`? `false` means SpongeAuth is required.

---

lets keep enabled for now, untill we figure out deployment for auth stuff